e2e/nomostest/config_sync.go

```diff
@@ -664,13 +664,14 @@ func setupDelegatedControl(nt *NT) {
 			continue
 		}
 		var rs *v1beta1.RootSync
-		switch source.(type) {
+		switch tSource := source.(type) {
 		case *syncsource.GitSyncSource:
-			rs = RootSyncObjectV1Beta1FromRootRepo(nt, id.Name)
+			rs = nt.RootSyncObjectGitSyncSource(id.Name, tSource)
+		// TODO: setup OCI & Helm RootSyncs
```

Comment:
I could, but @sdowell had asked me to leave them out, since they weren't actually reachable without ntops helpers. We compromised and left them commented, in case we want to add ntops for them in the future.

I may do that, but I think i've spent enough time on this e2e cleanup in the last few weeks. So maybe later.

Reply:
ack